Content.Shared/Buckle/SharedBuckleSystem.Interaction.cs

```diff
@@ -114,11 +114,14 @@ private void OnBuckleInteractHand(Entity<BuckleComponent> ent, ref InteractHandE
         if (args.Handled)
             return;
 
+        var canUnbuckle = false;
+
         if (ent.Comp.BuckledTo != null)
-            TryUnbuckle(ent!, args.User, popup: true);
+            canUnbuckle = TryUnbuckle(ent!, args.User, popup: true);
```

Comment:
```suggestion
            args. Handled = TryUnbuckle(ent!, args.User, popup: true);
```
just do it like this and remove canUnbuckle